CONTRIBUTING.md

```diff
@@ -214,8 +214,12 @@ When a person is assigned to an issue this indicates an _intent to implement_.
 
 * Include screenshots and animated GIFs in your pull request whenever possible.
 * List any steps necessary to trigger the feature you've created or bug you are fixing
-* Always run the [tests](#tests) locally before creating your PR
+* Always run the [unit tests](#unit-tests) locally before creating your PR
+ * You are not required to run the [integration tests](#integration-tests) locally
```

Comment:
Hmm, I like where you're going with that @jlongster. Might be nice to say something like

*we recommend running the unit tests locally.* and then say the integration tests can be run on CI automatically...